docs/dockerfile-frontend-options.md

```diff
@@ -0,0 +1,65 @@
+# Dockerfile Frontend Build Options
+
+## use with `buildctl build --opt`
```

Comment:
"This page documents the build options defined by Dockerfile frontend. When using `buildctl` these options can be passed when running `buildctl build --frontend dockerfile.v0 --opt ...` . When you are implementing your own frontend you should use the same option names to make sure equivalent `docker buildx`(link) CLI flags configure your frontend properly."